### ISSUE TITLE
build process fails because of SSL protocol version

### ISSUE THREAD

**Issue by maburg** [icaruseu/mom-ca#735]
we need a workaround until we upgrade to exist3.
Meanwhile storing resources locally?

**Comment by maburg** [comment]
BUILD FAILED 
/home/ubuntu/exist2/mom.XRX/build.xml:435: The following error occurred while executing this line: 
/home/ubuntu/exist2/mom.XRX/build.xml:184: javax.net.ssl.SSLException: Received fatal alert: protocol_version 

in Line 184 xquery resources are requested:   <get dest="tmp/jQuery.zip" src="https://github.com/jquery/jquery-ui/archive/1-9-stable.zip"/>

**Comment by maburg** [comment]
<get dest="tmp/jQuery.zip" src="https://github.com/jquery/jquery-ui/archive/1-9-stable.zip"/>

**Comment by maburg** [comment]
get dest="tmp/jQuery.zip" src="https://github.com/jquery/jquery-ui/archive/1-9-stable.zip"

**Comment by yngwi** [comment]
Aparrently there is a probleme with some websites and the ssl version used by Java 6. I tried around a bit but could not fix it yet. This is not a problem with Java 8 and eXist 3 so I just added a `vendor` directory to the code and added the zip files for jQuery-UI and CodeMirror as both cause the same problem. This way the build works again and we are independent from this kind of problem until the switch to eXist-3 is complete (working on it).
